src/Tribe/Tickets.php

```diff
@@ -30,63 +30,71 @@ class Tribe__Tickets__Tickets {
 		/**
 		 * Flag used to track if the registration form link has been displayed or not.
 		 *
-		 * @var boolean
+		 * @param boolean
 		 */
 		private static $have_displayed_reg_link = false;
 
 		/**
-		 * All Tribe__Tickets__Tickets api consumers. It's static, so it's shared across all child.
+		 * All Tribe__Tickets__Tickets api consumers. It's static, so it's shared across all children.
 		 *
-		 * @var array
+		 * @param array
 		 */
 		protected static $active_modules = array();
 
 		/**
+		 * Default Tribe__Tickets__Tickets ecommerce module.
+		 * It's static, so it's shared across all children.
+		 *
+		 * @param string
+		 */
+		protected static $default_module = 'Tribe__Tickets__RSVP';
+
+		/**
 		 * Indicates if the frontend ticket form script has already been enqueued (or not).
 		 *
-		 * @var bool
+		 * @param bool
```

Comment:
For our class properties `@var` is actually the correct doc tag - these aren't parameters (`@param` is generally only used for function/method args and for the variables passed via hooks).

[phpdoc.org notes for @var](https://phpdoc.org/docs/latest/references/phpdoc/tags/var.html)